rpi-eeprom-update

```diff
@@ -544,10 +549,22 @@ findBootFS()
       BOOTFS="${TMP_BOOTFS_MNT}"
    fi
 
+   # Some distros mount boot partition in alternative locations, e.g. Debian
+   # We can aid them by probing the most common alternative paths
+   if [ -n "${BOOTFS_ALTERNATIVE_PATHS}" ]; then
```

Comment:
Should this be in the `else` clause of the previous `if` block?

Also, if `$BOOTFS` (i.e. `/boot`) _does_ contain an `.elf` file, perhaps it should skip checking `$BOOTFS_ALTERNATIVE_PATHS`? :man_shrugging:

Reply:
I've addressed both, thanks!